components/panic_hook/lib.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! A simple panic hook that allows skipping printing stack trace conditionally.
+
+#![feature(box_syntax)]
```

Comment:
Could you remove it?

Reply:
Is there any drawbacks about using the box syntax?

Reply:
The syntax is for "box foo". It is not mandatory because it is just a syntax sugar for `Box::new`.